### DISCUSSION TITLE
XamlC error XFC0000: Cannot resolve type "PrimsApplication"

### DISCUSSION THREAD

**Discussion by ClayAchahui** [PrismLibrary/Prism#2319] [Xamarin.Forms]
How i can fix that...? I searched but nothing, people said "clear obj and bin folders" i tried it. Help... 
Xamarin forms 5.0.0.1905
![image](https://user-images.githubusercontent.com/43356162/105745693-8f19a180-5f0c-11eb-990d-4b344928227b.png)

**Comment by dansiegel** [comment]
If there are no references to Prism.DryIoc.Forms or Prism.Unity.Forms, XAMLC may not see the assembly which means it won't pick up the namespace and won't be able to properly link the aliased PrismApplication in XAML to the generated base type. If you get this error you simply adding the PrismApplication base class in Code can help resolve such an error.
